src/osd/PGLog.h

```diff
@@ -476,18 +476,14 @@ struct PGLog : DoutPrefixProvider {
     }
 
     void index(pg_log_dup_t& e) {
-      if (PGLOG_INDEXED_DUPS) {
```

Comment:
As Kefu's fix shows, this should be a check whether the flag is set in `indexed_data`.